molecule/default/tests/test_default.py

```diff
@@ -5,6 +5,19 @@
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
 
 
+def test_installed_packages(host):
+    assert host.package('filebeat').is_installed
+    assert host.package('filebeat').version == '6.6.1'
+
+
+def test_files(host):
+    to_add = [
+        '/etc/filebeat/filebeat.yml',
+    ]
+    for config in to_add:
+        assert host.file(config).exists
+
```

Comment:
In tests, this should be written like

```python
@pytest.mark.parametrize('config', [
    ('/etc/filebeat/filebeat.yml'),
])
def test_files(host, config):
    with host.sudo():
        assert host.file(config).exists
```

Reply:
Copied from sansible.logstash but updated